Lesson16/sinem16/script.js

```diff
@@ -438,17 +480,17 @@ console.log(`Words containing '${searchLetter}':`, filteredWords);
       - logs the popped element
       - logs the final array
 */
+console.log("Exp.24");
+
 function pushPopExample(arr, itemToPush) {
+  let pustItem = arr;
```

Comment:
Here, you are creating a second name tag (not a copy!) for the same array in memory.

If you wanted to keep the original safe, you need to clone it first: 
`let newItem = [...arr];`